pkgs/development/python-modules/keras/default.nix

```diff
@@ -7,10 +7,13 @@
 , pytest_xdist
 , six
 , Theano
+, tensorflow ? null
```

Comment:
in any case, we should have only one `tensorflow` parameter. When this function is called one can choose which `tensorflow` to pass.